### PR TITLE
.coafile: Enable RSTcheckBear 

### PR DESCRIPTION
**Pull request by mridubhatnagar**
Enable RSTCheckBear in the .coafile
present in the documentation repo.

Closes https://github.com/coala/documentation/issues/432

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -17,3 +17,7 @@ enabled = False
 files = **/*.rst, README.rst
 ignore = venv/**
 bears = InvalidLinkBear
+
+[rst]
+bears = RSTLintingBear, rstCheckBear
```
Comment (Vamshi99) [inline .coafile]: You have written names of these bears wrong, for correct names please checkout https://github.com/coala/bear-docs#restructuredtext

Comment (mridubhatnagar) [inline .coafile]: Should I make changes, and push the code. Then squash the commits?

File: .coafile [hunk 2]
```diff
@@ -17,3 +17,7 @@ enabled = False
...
+files = **/*.rst, README.rst
```
Comment (Vamshi99) [inline .coafile]: Also, change this to `files = *.rst, **/*.rst`

File: .coafile [hunk 3]
```diff
@@ -17,3 +17,7 @@ enabled = False
 files = **/*.rst, README.rst
 ignore = venv/**
 bears = InvalidLinkBear
+
+[rst]
+bears = ReSTLintBear, RSTcheckBear
```
Comment (Vamshi99) [inline .coafile]: You are wrong again, use `reSTLintBear`

Comment (mridubhatnagar) [inline .coafile]: oh okay

**Comment by mridubhatnagar** [comment]
Can someone look into this PR. And guide me how the fixes are to be made.

File: .coafile [hunk 4]
```diff
@@ -17,3 +17,7 @@ enabled = False
 files = **/*.rst, README.rst
 ignore = venv/**
 bears = InvalidLinkBear
+
+[rstcheck]
+bears = RSTcheckBear
+files = *.rst,**/*.rst
```
Comment (meetmangukiya) [inline .coafile]: if you are using both of this, isn't it equivalent to `**.rst`, check?

**Comment by meetmangukiya** [comment]
Remove `on this repo` from commit shortlog

File: .coafile (hunk 4 above)
Comment (mridubhatnagar) [inline .coafile]: yeah. Instead of this. If I do just only **/*.rst. Would mean it would check the directories as well as the sub-directories.

Comment (mridubhatnagar) [inline .coafile]: Thanks . **.rst is equivalent of both. Checked on command prompt. By importing glob.

**Comment by jayvdb** [comment]
Not the best commit message as https://github.com/coala/documentation/pull/438#issuecomment-300239804 was not done.
But, it isnt breaking the rules.

**Comment by jayvdb** [comment]
ack 91573d3

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**